data_set/taskset/sanity_check.md

```diff
@@ -0,0 +1,276 @@
+# Tellina sanity check
+Sanity check to ensure that the current version of Tellina can handle this representative set of tasks.
+
+A set of suggestions were tagged as 'helpful` if it contains the answer or a part of the answer. If the suggestions were nonsensical or unusable to solve the task, it was tagged as 'unhelpful'.
```

Comment:
Formatting error here. "'helpful\`" should be "'helpful'" (the little back tick)

Reply:
Oh wow! Thanks a lot for spotting that!